server/views/pages/order/attachments/view.njk

```diff
@@ -0,0 +1,82 @@
+{% extends "../../../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - Form Details" %}
+{% set mainClasses = "app-container govuk-body" %}
+{% from "moj/components/page-header-actions/macro.njk" import mojPageHeaderActions %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% block content %}
+
+{{ govukBackLink({
+  text: "Back",
+  href: "/order/" + order.id + "/summary"
+}) }}
+
+{% if order.status == "SUBMITTED" %}
+  {% set html %}
+    <p class="govuk-notification-banner__heading">
+      You are viewing a submitted order.
+    </p>
+  {% endset %}
+
+
+{{ govukNotificationBanner({
+  html: html
+}) }}
+{% endif %}
+
+<div class="homepage-content">
+  {{ mojPageHeaderActions({
+    heading: {
+        text: "Attach a document"
+    }
+  }) }}
+</div>
+ {% if order.status == "SUBMITTED" %}
+<span class="govuk-caption-l">Check your answers before clicking continue</span>
+{% endif %}
+
+<dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Licence
+    </dt>
+    {% if licenceFileName %}
+    <dd class="govuk-summary-list__value">
+       <a href="/order/{{order.id}}/attachments/licence/{{licenceFileName}}" class="govuk-link">{{licenceFileName}}</a>
+    </dd>
+    {% else %}  
+     <dd class="govuk-summary-list__value">
+       No licence document uploaded
+    </dd>   
+    {% endif %}
+    <dd class="govuk-summary-list__actions">
+      <a class="govuk-link"  href="/order/{{order.id}}/attachments/licence">Change<span class="govuk-visually-hidden"> name</span></a>
+    </dd>
+  </div>
+   <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Photo ID
+    </dt>
+    {% if photoFileName %}
+    <dd class="govuk-summary-list__value">
+     <a href="/order/{{order.id}}/attachments/photoId/{{photoFileName}}" class="govuk-link">{{photoFileName}}</a>
+    </dd>
+     {% else %}  
+     <dd class="govuk-summary-list__value">
+       No photo id document uploaded
+    </dd>   
+    {% endif %}
+    <dd class="govuk-summary-list__actions">
+      <a class="govuk-link" href="/order/{{order.id}}/attachments/photoId">Change<span class="govuk-visually-hidden"> name</span></a>
+    </dd>
+  </div>
+</dl>
+
```

Comment:
Should there be a next button / return to summary button

Reply:
added Back to form section button